content/docs/esc/integrations/_index.md

```diff
@@ -10,6 +10,7 @@ menu:
     weight: 6
 aliases:
     - /docs/esc/providers/
+    - /docs/pulumi-cloud/esc/providers/ 
```

Comment:
We don't want this here -- it should actually be an alias for the Dynamic Login Credentials page (`/docs/esc/integrations/dynamic-login-credentials`). I'll add it there also.

```suggestion
```

Reply:
Actually come to think of it, we link to this URL in a few different places in different ways. Probably better just to alias the top-level integrations page after all. Let's leave it!